developer_manual/design/icons-gen.py

```diff
@@ -31,7 +31,7 @@
 
 result = ""
 for icon, path in sorted(icons.items()):
-	result += ".. figure:: " + path[3:-3]+"*\n   :height: 32\n\n   " + icon[1:] + "\n\n"
+	result += ".. figure:: " + path[3:-3]+"*\n   :height: 32\n   :width: 32\n\n   " + icon[1:] + "\n\n"
```

Comment:
SOme icons are NOT 32x32 (breadcrumbs for example) so only the height was a good hack I thought :)

Reply:
Looked good here 🙈